minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/bootstrap/BootstrapListener.java

```diff
@@ -291,7 +293,7 @@ private void writeDump(OutputStream out) {
     private void echoRequestCmd(String cmd, OutputStream out) {
         try {
             out.write((cmd + "\n").getBytes(StandardCharsets.UTF_8));
-            out.flush();
+            out.close();
```

Comment:
Is it possible to lose this command if the buffer isn't flushed?

Reply:
Yeah it is possible I tried to look around and it is basically a no brainer that close should also do flush and it USUALLY does but it is not GUARANTEED. I added flush back just to be sure.